cloudpickle/cloudpickle.py

```diff
@@ -278,6 +278,20 @@ def __init__(self, file, protocol=None):
         # map ids to dictionary. used to ensure that functions can share global env
         self.globals_ref = {}
 
+        # When unpickling a function created in a dynamic module or the __main__
+        # module, its set of global variables may be different from the one from
+        # the module the function belongs to, for example if the dynamic module
+        # has had its variables mutated between pickling and unpickling time
```

Comment:
I find this misleading. I would say something like:
```suggestion
        # has had a global variable mutated between two call to pickle.dump,
        # then the two pickled objects have different value for the same variable.
```